pkgs/top-level/python-packages.nix

```diff
@@ -18315,17 +18315,21 @@ in modules // {
   };
 
   sopel = buildPythonPackage rec {
-    name = "sopel-6.1.1";
+    name = "sopel-6.2.1";
 
     src = pkgs.fetchurl {
       url = "https://pypi.python.org/packages/source/s/sopel/${name}.tar.gz";
-      sha256 = "0nr2a88bkxg2593dd947qkh96g8j32q7pl7x9zvx4158h4bgx99y";
+      sha256 = "06m5clmg9x0bsnhvl5d75mskwqnxvkdd00p0dqnpwip9vmq6n8cz";
     };
 
+    buildInputs = with self; [ pytest ];
     propagatedBuildInputs = with self; [ praw xmltodict pytz pyenchant pygeoip ];
 
     disabled = isPyPy || isPy26 || isPy27;
 
+    checkPhase = ''
+    ${python.interpreter} test/*.py
+    '';
```

Comment:
Generally, when we have pytest, we invoke the test runner using
```
checkPhase = ''
  pytest test
```
Sometimes the default `${python.interpreter} test` is sufficient. I just checked upstream to find out how they run their tests, and I noticed this [file](https://github.com/sopel-irc/sopel/blob/6.2.1/pytest_run.py). Maybe we should run that instead in the checkPhase?